Servers/UI/OJS.Servers.Ui/ClientApp/src/pages/contests/contest-solution-submit/ContestSolutionSubmitPage.tsx

```diff
@@ -148,6 +149,29 @@ const ContestSolutionSubmitPage = () => {
         });
     };
 
+    useEffect(() => {
+        if (submissionsData?.items && problems && submissionsData.items.length > 0) {
+            // eslint-disable-next-line max-len
+            const latestSubmission = submissionsData.items.reduce((newest, current) => new Date(current.createdOn) > new Date(newest.createdOn)
+                ? current
+                : newest, submissionsData.items[0]);
+
+            if (latestSubmission?.problem) {
+                const probIndex = problems.findIndex((p) => p.id === latestSubmission.problem.id);
+                if (probIndex !== -1) {
+                    const updatedProb = { ...problems[probIndex] };
```

Comment:
We generally don't like abbreviations and encourage full names for variables -> `updatedProblem`

Reply:
Done.